ext/dom/tests/DOMDocument_loadHTML_error2.phpt

```diff
@@ -14,4 +14,4 @@ try {
 }
 ?>
 --EXPECT--
-DOMDocument::loadHTML(): Argument #1 ($source) must not be empty
+DOMDocument::loadHTML(): Argument #1 ($source) cannot be empty
```

Comment:
Didn't we switch some messages to "must not" in the past because "cannot" can be ambiguous, depending on the context? Maybe @kocsismate can clarify.

Reply:
That might be the case, but the most common wording at the moment is `cannot be empty`.

Having the helper function earlier would probably have flagged this. :/

Reply:
Found this reply: https://github.com/php/php-src/pull/10917#discussion_r1150110046

Reply:
I would also prefer that this is unified in favor of `must …`, because that matches the wording for values outside of a given range (e.g. `must be greater than or equal to 0`).

Reply:
Agreed with Tim

Reply:
I pushed a new commit using a sed command for the find and replace